src/ui/components/Transactions.tsx

```diff
@@ -2,62 +2,54 @@
 // SPDX-License-Identifier: GPL-3.0-only
 
 import { BellIcon, XIcon } from '@heroicons/react/outline';
-import React from 'react';
 import { NotificationIcon } from './common/NotificationIcon';
-import type { TransactionsState } from 'types';
+import type { QueuedTxOptions, TransactionsState } from 'types';
 import { classes, isEmptyObj } from 'ui/util';
 
 export function Transactions({
   dismiss,
   txs,
 }: React.HTMLAttributes<HTMLDivElement> & TransactionsState) {
-  const Notifications: JSX.Element[] = [];
-  for (const id in txs) {
-    const { extrinsic, status, events } = txs[id] || {};
-    const isComplete = status === 'error' || status === 'success';
+  return (
+    <div className={classes('z-10 fixed right-3 top-3 w-80')}>
```

Comment:
what is the reason behind this change?

Reply:
It was to remove the spread operator which was not allowed by the default vite settings and also it makes more sense to use reduce in my opinion.